### DISCUSSION TITLE
Run CoastalApp with a wind that is given on a mesh that is part of the mesh that ADCIRC-WW3 uses

### DISCUSSION THREAD

**Discussion by sarriegi** [noaa-ocs-modeling/CoastalApp#182] [Q&A]
Hello, have you ever tried to run CoastalApp using a wind forcing that is given on a mesh that is part of the mesh that the model uses to run? For example, ADCIRC-WW3 runs on a mesh that includes a big part of the North Atlantic, but the wind is given only on part of this mesh, as shown below:
![image](https://github.com/noaa-ocs-modeling/CoastalApp/assets/77630268/64cd536b-55f9-412c-905c-e992739356c6)
Would this work in your opinion? I tried but I got errors right at the start.
@pvelissariou1 @saeed-moghimi-noaa 
Angelos

**Comment by pvelissariou1** [comment]
Hi Angelos,
No, unfortunately you need to have the wind data for the whole domain you
are using. You may set the wind to zero for the missing fields, or you have
to use a subsetted mesh for your simulations.

On Monday, January 29, 2024, sarriegi ***@***.***> wrote:
> Hello, have you ever tried to run CoastalApp using a wind forcing that is
given on a mesh that is part of the mesh that the model uses to run? For
example, ADCIRC-WW3 runs on a mesh that includes a big part of the North
Atlantic, but the wind is given only on part of this mesh, as shown below:
> image.png (view on web)
> Would this work in your opinion? I tried but I got errors right at the
start.
> @pvelissariou1 @saeed-moghimi-noaa
> Angelos
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.<
https://ci6.googleusercontent.com/proxy/Myw24PrQBgLqLmI9iPUpZmWwXFJiLq1L-oCWl6RqVqvBolXclFepMuEAh8jZLxjgWJujYlgZvK6o59-Zkr3BKzJBAKwIuezmdSVx4PszCqyJ9PbTd99Fo8Lh0LPrfoyA88bmtKDKFyrO7tVVIOaHClm-Lxy4VxA_x8Q5afVAzhwzoaHndjPEUY9FWjDhrUg90sSnQQvIpvVJhrKDwWrMiYPTu4c4p0d1_NfTLmcjUQBJUQ=s0-d-e1-ft#https://github.com/notifications/beacon/APC7TPYM4DSKPDMGJ7M56ODYRAJRDA5CNFSM6AAAAABCQFFDRKWGG33NNVSW45C7OR4XAZNKIRUXGY3VONZWS33OVJRW63LNMVXHIX3JMTHAAXO3HU.gif>Message
ID: ***@***.***>

-- 
Panagiotis Velissariou, Ph.D., P.E.
Scientist III
OAI at the Office of Coast Survey CSDL/CMMB
National Ocean Service
National Ocean and Atmospheric Administration
cell: (205) 227-9141
email: ***@***.***
